sdk/documentintelligence/ai-document-intelligence-rest/src/pollingHelper.ts

```diff
@@ -51,10 +50,6 @@ export interface SimplePollerLike<TState extends OperationState<TResult>, TResul
    * Returns the state of the operation.
    */
   getOperationState(): TState;
-  /**
-   * Returns the id of the operation.
-   */
-  getOperationId(): string;
   /**
    * Returns the result value of the operation,
    * regardless of the state of the poller.
```

Comment:
I can't comment on the specific lines but the poller's `TState` should be of type `AnalyzeBatchOperationOutput` not `OperationState`

Reply:
After the offline discussion, landed on `OperationState<TResult>`, so no changes.

Ideally should have been AnalyzeBatchOperationOutput, but since the operation status type has mismatch, we felt it is not worth changing too many things and rely on `updateState`